README.md

```diff
@@ -30,6 +30,7 @@ The game (not the studio) also can be launched without any added arguments if St
 ## Tips
  * Be advised to set InterPupillary Distance (IPD) in the settings to change the scale of the world according to own taste and used hardware.
  * If VR mode doesn't launch, make sure that neither of controllers is asleep during game launch.
+ * For best compatibility use together with HongFire patch [KK-HF_Patch](https://github.com/ManlyMarco/KK-HF_Patch) [KKS-HF_Patch](https://github.com/ManlyMarco/KKS-HF_Patch).
```

Comment:
```suggestion
 * For best compatibility use together with HF Patch [KK-HF_Patch](https://github.com/ManlyMarco/KK-HF_Patch) [KKS-HF_Patch](https://github.com/ManlyMarco/KKS-HF_Patch).
```
The HF name no longer points specifically to the lost forums. If anything, it's "Have Fun Patch" now.